qa/tests/features/new/manual/filter-refiner/Diagram/Diagram.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Filter refiner, Diagrams
+
+Background: 
+	Given user opens the "Filter Refiner" for the "PGP3140_wgs_panel_hl" dataset
+
+@regression
+Scenario Outline: 01 Expand the "<Attribute Name>" diagram for attribute with less than 40 values 
+	When user clicks the down arrow (v) button near "<Attribute Name>" in the left part of the page
+	Then the diagram for "<Attribute Name>" should be expanded
+	And the column number should equal "<Number Of Columns>"
+	And the diagram should correspond to attribute by the value's variant number
+
+Examples: 
+| Attribute Name       | Number Of Columns |
+| Callers              | 10                |
+| Canonical_Annotation | 18                |
```

Comment:
***Please add starting tabulation for lines inside "Examples" table to make the text more readable